taichi/lang_util.h

```diff
@@ -109,6 +109,22 @@ inline bool constexpr is_unsigned(DataType dt) {
   return !is_signed(dt);
 }
 
+inline DataType to_unsigned(DataType dt) {
+  TI_ASSERT(is_signed(dt));
+  switch (dt) {
+    case DataType::i8:
+      return DataType::u8;
+    case DataType::i16:
+      return DataType::u16;
+    case DataType::i32:
+      return DataType::u32;
+    case DataType::i64:
+      return DataType::u32;
```

Comment:
Why do we use u32 here. Is u64 better?
```suggestion
      return DataType::u64;
```

Reply:
Thanks for catching this! there's no reason to use u32 here.